framework/test/mocha/unit/modules/chain/transport.js

```diff
@@ -23,21 +23,19 @@ const {
 } = require('@liskhq/lisk-transactions');
 const { transfer, TransactionError } = require('@liskhq/lisk-transactions');
 
-const accountFixtures = require('../../../../fixtures/accounts');
-const Bignum = require('../../../../../../src/modules/chain/helpers/bignum');
-const Block = require('../../../../fixtures/blocks').Block;
+const accountFixtures = require('../../../fixtures/accounts');
+const Bignum = require('../../../../../src/modules/chain/helpers/bignum');
+const Block = require('../../../fixtures/blocks').Block;
 const {
 	registeredTransactions,
-} = require('../../../../common/registered_transactions');
-const InitTransaction = require('../../../../../../src/modules/chain/logic/init_transaction');
-const ProcessTransactions = require('../../../../../../src/modules/chain/submodules/process_transactions');
-const processTransactionLogic = require('../../../../../../src/modules/chain/logic/process_transaction');
+} = require('../../../common/registered_transactions');
+const InitTransaction = require('../../../../../src/modules/chain/logic/init_transaction');
+const ProcessTransactions = require('../../../../../src/modules/chain/submodules/process_transactions');
```

Comment:
`ProcessTransactions` is really not a good name for a module or a class. It should be renamed to `TransactionsProcessor` or `TransactionsLoader`

Reply:
This will be change in coming PRs